bin/compile

```diff
@@ -55,10 +59,12 @@ unset RECEIVE_DATA RUN_KEY BUILD_INFO DEPLOY LOG_TOKEN
 unset CYTOKINE_LOG_FILE GEM_PATH
 
 # Syntax sugar.
-source $BIN_DIR/utils
+# shellcheck source=/dev/null
```

Comment:
Why this, and not the correct path? That's what the comment is for, and you want the import to occur, so that Shellcheck can follow it